walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -15,9 +15,76 @@
 package com.google.sps;
 
 import java.util.Collection;
+import java.util.Collections;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Set;
 
 public final class FindMeetingQuery {
-  public Collection<TimeRange> query(Collection<Event> events, MeetingRequest request) {
-    throw new UnsupportedOperationException("TODO: Implement this method.");
-  }
-}
+    public Collection<TimeRange> query(Collection<Event> events, MeetingRequest request) {
+        Collection<TimeRange> timeRanges = new ArrayList<TimeRange>();
+        List<TimeRange> attendeeRelevantTimeRanges = new ArrayList<TimeRange>();
+        Collection<String> requestAttendees = request.getAttendees(); 
+        Collection<String> eventAttendees;
+
+        // Add the TimeRanges of events into a List that are relevant to the request's attendees 
+        for(Event event : events){
+	        eventAttendees = event.getAttendees(); 
+	        if(!Collections.disjoint(requestAttendees, eventAttendees)){
+	            attendeeRelevantTimeRanges.add(event.getWhen());
```

Comment:
Nit: indentation

Reply:
The indentation seems fine on my end. Unsure how to fix indentation.